packages/mobile/src/components/fields/SwitchRowField.tsx

```diff
@@ -0,0 +1,29 @@
+import { useField } from 'formik'
```

Comment:
Was the motivation for moving this out of the edit-track-screen just to make it more generalized? Or match web? 
0 issues with it but just curious what the motivation was

Reply:
yeah it was to match web's location of switch-row-field. in fact im going to update some existing implementations on mobile that reimplement the above ^ in edit album and edit track